app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>This is from post page</h1>
\ No newline at end of file
+<div class="container">
+  <div class="row grid gap-3">
+    <div class="col-6 col-md-4 border border-3 border-black">
+      <img src="<%= @user.photo %>" />
+    </div>
+    <div class="col-6 col-md-8 border border-3 border-black">
+      <h2> <%= @user.name %></h2>
+      <sub>Number of posts <%= @user.posts.size %></sub>    
+    </div>
+  </div>
+  <div class="row">
+    <% @posts.each_with_index do |post, index| %>
+    <div class="col col-12 border border-3 border-black">
+      <span>Post # <%= index+1 %></span>
+        <p><%= post.text %></p>
+        <sub>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %> </sub>
+    </div>  
+    <div class="col col-12 border border-3 border-black">
+        <span>Comments</span>
+        <% post.comments.each do |comment| %>
+        <p><%= comment.text %></p>
+        <% end %>
+    </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] On the user posts page you should render the user's 3 most recent post and i suggest that you add `See all Post` link to be able to view all posts of the user

Reply:
- [ ] One post should render 5 most recent comments

Reply:
Thank you for the feedback, I will do that